File: rb/utility/make_context.rb
# Bng2latlongConverter SDK utility: make_context
require_relative '../core/context'
module Bng2latlongConverterUtilities
  MakeContext = ->(ctxmap, basectx) {
    Bng2latlongConverterContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// Bng2latlongConverter SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class Bng2latlongConverterMakeContext
{
    public static function call(array $ctxmap, ?Bng2latlongConverterContext $basectx): Bng2latlongConverterContext
    {
        return new Bng2latlongConverterContext($ctxmap, $basectx);
    }
}
